xmake/rules/c++/modules/build_modulefiles.lua

```diff
@@ -35,7 +35,7 @@ function _build_modulefiles_clang(target, sourcebatch, opt)
     end
 
     -- compile module files to *.pcm
-    opt = table.join(opt, {configs = {force = {cxxflags = {"-fmodules-ts", "--precompile", "-x c++-module"}}}})
+    opt = table.join(opt, {configs = {force = {cxxflags = {"-fmodules", "--precompile", "-x c++-module"}}}})
```

Comment:
should use opt.modulesflag

Reply:
This is a commented out portion of the code that I left commented. The changes are due to the use of replace and find the code. Please take a look at the other pull request.